### PR TITLE
Update progress.v1.yaml

### PR DESCRIPTION
**Pull request by mcginkel**

### CODE REVIEW

File: reference/progress.v1.yaml [hunk 1]
```diff
@@ -312,316 +162,18 @@ paths:
...
 components:
   schemas:
-    Statement:
-      title: XAPI Statement
+    SimpleProgress:
+      title: SimpleProgress
       type: object
       x-examples:
         example-1:
           id: string
```
Comment (hhaarsma) [inline reference/progress.v1.yaml:170]: Is there a usefull value for this? Shouldnt we use the combination of (Activity/AU)/Student

File: reference/progress.v1.yaml [hunk 2]
```diff
@@ -312,316 +162,18 @@ paths:
...
-              raw: 0
-              min: 0
-              max: 0
-            success: true
-            completion: true
-            response: string
-            duration: string
-            extensions: {}
-          context: {}
+          activityId: string
```
Comment (hhaarsma) [inline reference/progress.v1.yaml:171]: Should we use AssignableUnitId here, to make clear that this is the ID of the AU?

File: reference/progress.v1.yaml [hunk 3]
```diff
@@ -312,316 +162,18 @@ paths:
...
+          studentEckId: string
+          studentId: string
+          status: not-started
           timestamp: string
-          stored: string
-          authority: {}
           version: string
```
Comment (hhaarsma) [inline reference/progress.v1.yaml:176]: What info do we put here? Is this just to keep the correct order of events, is timestamp then enough?

File: reference/progress.v1.yaml (hunk 1 above)
Comment (mcginkel) [reply reference/progress.v1.yaml:170]: Ik zou een gui verwachten. kun je later kijken of je deze event/record al geprocessed hebt.

File: reference/progress.v1.yaml (hunk 3 above)
Comment (mcginkel) [reply reference/progress.v1.yaml:176]: i have updated it to schemaVersion

**Review by hhaarsma — APPROVED** [review]

**Pull request merged**